docs/getting-started/configuration.md

```diff
@@ -80,14 +80,26 @@ superbench:
       parameters:
         <<: *common_model_config
         batch_size: 128
-    cnn_models:
+    resnet_models:
```

Comment:
Why change this?

Reply:
because we changed default config previously, changed accordingly here in docs